python/lib/cloudutils/utilities.py

```diff
@@ -216,8 +216,8 @@ def isKVMEnabled(self):
 class serviceOpsRedhat7(serviceOps):
     def isServiceRunning(self, servicename):
         try:
-            o = bash("systemctl status " + servicename)
-            if "running" in o.getStdout() or "start" in o.getStdout() or "Running" in o.getStdout():
+            o = bash("systemctl is-active " + servicename)
+            if "inactive" not in o.getStdout():
```

Comment:
if statement can be simplified to: `return "inactive" not in o.getStdout()`

Reply:
Simplified, fixed. Thanks.